.mvn/mvnd.properties

```diff
@@ -1,19 +1,3 @@
-#
```

Comment:
This is for mvnd, so a rather custom config file, not something that's been generated.

Reply:
just to make sure: i should just revert the change in this file, but no longer making a license header check below `.mvn` folder you agree with?

Reply:
yep